host.json

```diff
@@ -13,6 +13,10 @@
       }
     }
   },
+  "extensionBundle": {
+    "id": "Microsoft.Azure.Functions.ExtensionBundle",
+    "version": "[2.*, 3.0.0)"
```

Comment:
this can be safely be set to `[3.x,4.0.0)`

Reply:
resolved in https://github.com/pagopa/io-functions-assets/pull/69/commits/90606f50a60c08dbeeb0560d412753e7abb9719f